packages/fxa-auth-server/lib/senders/emails/layouts/fxa/en.ftl

```diff
@@ -4,7 +4,10 @@
 ## version. The strings are usually identical but sometimes they differ slightly.
 
 fxa-header-firefox-logo = <img data-l10n-name="fxa-logo" alt="{ -brand-firefox } logo">
+fxa-header-mozilla-logo = <img data-l10n-name="mozilla-logo" alt="{ -brand-mozilla } logo">
 fxa-header-sync-devices-image = <img data-l10n-name="sync-devices-image" alt="Sync devices">
 body-devices-image = <img data-l10n-name="devices-image" alt="Devices">
 fxa-privacy-url = { -brand-mozilla } Privacy Policy
+moz-accounts-privacy-url = { -product-mozilla-accounts(capitalization:"uppercase") } Privacy Policy
```

Comment:
@dschom @LZoog - Example of parametrization in ftl strings to pick between two different options of the same term (set up in `branding.ftl`)

Based on "parametrized terms" section of: 
https://projectfluent.org/fluent/guide/terms.html